two-phase-commit/src/tpc_master.c

```diff
@@ -0,0 +1,143 @@
+/* 2PC master node */
+
+#include <pthread.h>
+#include <unistd.h>
+#include "tpc.h" /* generated by rpcgen */
+#include "vote.h"
+#include "tpc_master.h"
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+int *results;
+CLIENT **cl_arr;
+char **args;
+
+typedef struct thread_arg {
+    pthread_t tid;
+    commit_vec_args *commit_args;
+} thread_arg;
+
+int main(int argc, char *argv[])
+{
+    if (argc < 2) {
+        printf("Usage:\n");
+        return 1;
+    }
+
+    // argv[1..n] are the IP addresses of the slave nodes.
+    int num_slaves = argc - 1;
+    args = argv;
+
+    results = (int*) malloc(num_slaves * sizeof(int));
```

Comment:
Very not-critical, but the spacing in type casts is not consistent. The inconsistencies occur both before the asterisks and after the parentheses.

Reply:
first line of last code sample at https://www.gnu.org/prep/standards/html_node/Syntactic-Conventions.html#Syntactic-Conventions shows how I'll fix it.